WordPress/gutenberg
# [RNMobile] Audio block - UI fixes

`gutenberg mobile PR` https://github.com/wordpress-mobile/gutenberg-mobile/pull/3118
`WP Android` https://github.com/wordpress-mobile/WordPress-Android/pull/14007



## Description

Hides the replace audio button once an upload is in progress. 

## How has this been tested?
To test:

1. Add a new post. 
2. Add an Audio block. 
3. Add Audio. 
4. Notice the replace audio button is not visible during upload. 

## Screenshots 

<kbd><img src="https://user-images.githubusercontent.com/1509205/107319183-46192f80-6a6c-11eb-90b4-a4ef761258d1.png" width="320"></kbd>
## Types of changes





## Checklist:
